audio/cmus/Portfile

```diff
@@ -3,19 +3,21 @@
 PortSystem 1.0
 PortGroup           github 1.0
 
-github.setup        cmus cmus 2.7.1 v
-revision            4
+github.setup        cmus cmus 2.8.0 v
+revision            0
 categories          audio
-maintainers         {g5pw @g5pw} openmaintainer
+maintainers         {g5pw @g5pw} \
+                    {Traace @Traace} \
+                    openmaintainer
 license             GPL-2+
 description         ncurses based music player
 long_description    cmus is a small and fast text mode music player for Linux \
                     and many other UNIX like operating systems.
 homepage            https://cmus.github.io/
 platforms           darwin
 
-checksums           rmd160  3a9ff85f3279ff74af5d46cca2d8bf5b79671c6b \
-                    sha256  56bbdb487cf15285dea630a71e31b094276e143cb4c15c411b1b3549ec6f23b3
+checksums           rmd160  81091c678f3534d006b77f1fe9621ae05022f028 \
+                    sha256  7986b1d86749c7f20f853cd6b35ffc13aac1c9d4d24cc5480292da0d904962d9
```

Comment:
The `size` parameter is still missing here and needs to be added.

Reply:
So did the macports committer g5pw forgot the size parameter before.

A newcomer should add it, thats okay. but why is he committer then, when newcomers are expected to do it better ? ...

Reply:
When this port was last updated, we weren’t in the habit of adding the size parameter, but now it’s expected. That’s why the difference (but you could not know that).